.github/workflows/CI.yml

```diff
@@ -0,0 +1,32 @@
+name: CI
+on: [ push, pull_request ]
+jobs:
+  Build:
+
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+        node-version: [ 10.x ]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Cache Node.js modules
+      uses: actions/cache@v2
+      with:
+        # npm cache files are stored in `~/.npm` on Linux/macOS
+        path: ~/.npm
+        key: ${{ runner.OS }}-node-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.OS }}-node-
+          ${{ runner.OS }}-
+    - name: Install ajv-cli
+      run: npm -g i ajv-cli
+    - name: Validate Plugins JSON File
+      run: ajv test -s resources/schema.json -d plugins.json --errors=text --valid
```

Comment:
Consider renaming schema to `plugins-schema.json` or something.

Reply:
Done 👍